packages/redux/src/themes/actions/factories/fetchThemesFactory.ts

```diff
@@ -0,0 +1,50 @@
+import * as actionTypes from '../../actionTypes.js';
+import {
+  type Config,
+  type GetThemes,
+  type QueryThemes,
+  type Themes,
+  toBlackoutError,
+} from '@farfetch/blackout-client';
+import type { Dispatch } from 'redux';
+
+/**
+ * Fetch styleguide themes.
+ *
+ * @param getThemes - Fetch styleguide themes client.
+ *
+ * @returns Thunk factory.
+ */
+const fetchThemesFactory =
+  (getThemes: GetThemes) =>
+  (code: string, query?: QueryThemes, config?: Config) =>
+  async (dispatch: Dispatch): Promise<Themes> => {
+    try {
+      dispatch({
+        type: actionTypes.FETCH_THEMES_REQUEST,
+        meta: { code },
+      });
+
+      const result = await getThemes(code, query, config);
+
+      dispatch({
+        type: actionTypes.FETCH_THEMES_SUCCESS,
+        payload: { result },
```

Comment:
```suggestion
        payload: result,
```

What about using the result directly as the payload? 🤔

Reply:
Done